t/rose-task-run/38-app-bunch-counts/app/cutoff/bin/echoer

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Nope, well spotted. In fact the whole bin/ directory under cutoff can be removed.